unit_tests/actuator/UnitTestActuatorBulk.C

```diff
@@ -0,0 +1,74 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+
+#include <gtest/gtest.h>
+#include <actuator/ActuatorBulk.h>
+#include <actuator/ActuatorInfo.h>
+
+// to allocate need turbine info
+// compute offsets need num procs
+// add fields/fixed fields
+// allocate
+namespace sierra{
+namespace nalu{
+
+namespace{
+
+TEST(ActuatorMeta, constructor){
+  const int numTurbines = 2;
+  ActuatorMeta fieldMeta(numTurbines);
+  EXPECT_EQ(numTurbines, fieldMeta.numberOfActuators_);
+  EXPECT_EQ(0, fieldMeta.num_points_turbine(0));
+  EXPECT_EQ(0, fieldMeta.num_points_turbine(1));
+}
+
+TEST(ActuatorMeta, addTurbine){
+  const int numTurbines = 1;
+  ActuatorMeta fieldMeta(numTurbines);
+  ActuatorInfoNGP dummyInfo;
+  dummyInfo.numPoints_=1024;
+  fieldMeta.add_turbine(dummyInfo);
+  EXPECT_EQ(1024, fieldMeta.num_points_turbine(0));
+}
+
+TEST(ActuatorMeta, copyCtor){
```

Comment:
Nitpick: Decide on 

```C++
void function_definition_1 {
  // ...
}

// vs

void function_definition2()
{
  // ...
}
```

Preferably run `clang-format`. Rest of nalu uses the latter

Reply:
sure.  I will run the formatter.